procmonAnalyze.js

```diff
@@ -500,8 +500,16 @@ async function readFileContents() {
     let profilerData = null;
     if (profilerInput.files[0]) {
       let profilerText = await getFileText(reader, profilerInput.files[0]);
+      let processStartTimes = data.filter(row => row.operation == "Process Start");
+      let processStartTime = processStartTimes[0].start;
+      for (let i = 0; i < processStartTimes.length-1; i++) {
+        if (processStartTimes[i+1].detail.includes("-contentproc")){
+          break;
+        } else {
+          processStartTime = processStartTimes[i+1].start;
+        }
+      }
```

Comment:
```suggestion
      let firstContentIndex = processStartTimes.findIndex(row => row.detail.includes("-contentproc"));
      let mainProcessIndex = firstContentIndex > 0 ? firstContentIndex - 1 : 0;
      let processStartTime = processStartTimes[mainProcessIndex];
```

Reply:
I'm not sure how these github code suggestions are supposed to work... looks like I can just commit it here? Trying that.